lib/frisby.js

```diff
@@ -1208,8 +1208,8 @@ function _jsonContains(jsonBody, jsonTest) {
 
     // Ensure property exists
     if(jsonBody) {
-      var kt = typeof jsonTest[key];
-      if(kt === "object") {
+      var kt = _toType(jsonTest[key]);
+      if(kt === "object" || kt === "array") {
```

Comment:
With this implementation there would need to be a matcher element for each corresponding element of the array, which might become unwieldy for large arrays. This change is pretty old so I've lost some confidence in it, and I would want to see more tests around the array scenarios. I'm thinking it would probably be better to close this PR and open an issue for Array support (if that is a thing people would like to see), rather than trying to jam support for Arrays in with a bug fix.